cmd/antrea-agent/options.go

```diff
@@ -273,3 +276,18 @@ func (o *Options) validateFlowExporterConfig() error {
 	}
 	return nil
 }
+
+func (o *Options) validateFlexibleIPAMConfig() error {
+	if features.DefaultFeatureGate.Enabled(features.FlexibleIPAM) {
+		if o.config.OVSDatapathType != string(ovsconfig.OVSDatapathSystem) {
+			return fmt.Errorf("FlexibleIPAM requires 'system' OVSDatapathType")
```

Comment:
Why it does not work for userspace datapath>?

Reply:
This method does not work on my testbed, and I cannot find the prerequisite from doc/comments, thus disable this method first.

Reply:
Even we could not figure out the exact reason now, at least add some comments to explain we found it does work and will rootcause later.

Reply:
Discussed with @antoninbas , 'netdev' requires another OVS bridge, which is not compatible with current AntreaIPAM data plane design. Added comments here.